starter/slack-spring-boot/src/main/kotlin/io/olaph/slack/broker/exception/ExceptionChain.kt

```diff
@@ -0,0 +1,12 @@
+package io.olaph.slack.broker.exception
+
+class ExceptionChain constructor(private val listOFErrors: MutableList<Throwable> = mutableListOf()) {
```

Comment:
Can you put kdoc on this class?

Reply:
no since documentation is only for idiots who can't read code